src/DataTable/Response/UserListResponse.php

```diff
@@ -0,0 +1,108 @@
+                    $roles = UserRoleEnum::ROLE_USER;
+                }
+
+                if ($this->decisionManager->decide($token, [UserRoleEnum::ROLE_TRANSLATOR])) {
+                    $roles .= ', '.UserRoleEnum::ROLE_TRANSLATOR;
+                }
+            }
+            $roles = ucfirst(strtolower(str_replace('ROLE_', '', $roles)));
+
+            $this->values['firstname'] = $user->getFirstname();
+            $this->values['lastname'] = $user->getLastname();
+            $this->values['email'] = $user->getEmail();
+            $this->values['semester'] = $semester;
+            $this->values['university'] = (null !== $user->getUniversity()) ? $user->getUniversity()->getName() : '';
+            $this->values['languages'] = $languages;
+            $this->values['languagesWanted'] = $languagesWanted;
+            $this->values['local'] = ($user->isLocal()) ? $this->translator->trans('yes') : $this->translator->trans('no');
+            $this->values['arrival'] = (null != $user->getArrival()) ? $user->getArrival()->format('d/m/Y') : '';
+            $this->values['createdAt'] = $user->getCreatedAt()->format('d/m/Y');
+            $this->values['nbBuddies'] = $user->getNbBuddies();
+            $this->values['nbBuddiesWanted'] = $user->getNbBuddiesWanted();
+            $this->values['nbReport'] = $user->getNbReport();
+            $this->values['nbWarned'] = $user->getNbWarned();
+            $this->values['nbBuddiesRefused'] = $user->getNbBuddiesRefused();
+            $this->values['nbBuddiesRemoved'] = $user->getNbBuddiesRemoved();
+            $this->values['isProfileCompleted'] = ($user->isReadyForMatching()) ? $this->translator->trans('yes') : $this->translator->trans('no');
+            $this->values['archive'] = ($user->getArchive()) ? $this->translator->trans('yes') : $this->translator->trans('no');
+            $this->values['roles'] = $roles;
+            $this->values['section'] = $user->getSection()->getName();
+            $this->values['actions'] = $this->templating->render('Default/listActions.html.twig', [
+                'urlEdit' => $this->router->generate('user_edit', ['id' => $user->getId()]),
+                'reportClass' => ($user->getNbReport() > 0) ? 'user_report' : null,
+                'urlMatch' => (!$user->isLocal() && $user->getNbBuddies() < $user->getNbBuddiesWanted())
+                               ? $this->router->generate('matching_step2', ['internationalId' => $user->getId()])
+                               : null,
+                'deleteClass' => ($user->getId() !== $user->getId()
+                    && !$this->decisionManager->decide($token, [UserRoleEnum::ROLE_SUPER_ADMIN]))
+                    ? 'user_delete'
+                    : null,
+            ]);
+            $this->values['DT_RowAttr'] = ['data-id' => $user->getId()];
+
+            $this->values['compatibility'] = (\array_key_exists('userToCompare', $options) && $options['userToCompare'] instanceof User)
```

Comment:
```suggestion
            $this->values['compatibility'] = (\array_key_exists('userToCompare', $options) && $options['userToCompare'] instanceof User)
```suggestion
            $this->values['compatibility'] = \array_key_exists('userToCompare', $options) && $options['userToCompare'] instanceof User
```

Reply:
done